### PR TITLE
feat(self-mailers) Add Self Mailers support

### PR DESCRIPTION
**Pull request by adrianfalleiro**
**What**
Adds support to this library to create, list, get, and cancel self mailers.

**Why**
So customers can interact with the Self Mailers API with the Java client.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,13 @@ Lob.init("yourApiKey", "yourApiVersion");
     - [Delete a Bank Account](https://lob.com/docs/java#bankaccounts_delete)
     - [Verify a Bank Account](https://lob.com/docs/java#bankaccounts_verify)
     - [List all Bank Accounts](https://lob.com/docs/java#bankaccounts_list)
+- **Self Mailers API**
+  - [Self Mailers](https://lob.com/docs/php#self_mailers)
```
Comment (senechko) [inline README.md:135]: php -> java

File: src/test/java/com/lob/model/SelfMailerTest.java [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+
+        LobResponse<SelfMailer> response = new SelfMailer.RequestBuilder()
+                .setDescription("Test Self Mailer")
+                .setOutside("<h1>Outside</h1>")
+                .setInside("<h1>Hello {{name}}</h1>")
+                .setMergeVariables(mergeVariables)
+                .setTo(
+                        new Address.RequestBuilder()
+                                .setCompany("Lob.com")
+                                .setLine1("185 Berry St Ste 6100")
```
Comment (senechko) [inline src/test/java/com/lob/model/SelfMailerTest.java:79]: our new offics is `210 King St`

File: README.md (hunk 1 above)
Comment (adrianfalleiro) [reply README.md]: d'oh. fixed

File: src/test/java/com/lob/model/SelfMailerTest.java (hunk 2 above)
Comment (adrianfalleiro) [reply src/test/java/com/lob/model/SelfMailerTest.java:79]: Updated, thanks

**Review by senechko — APPROVED** [review]

**Pull request merged**